roles/libvirt_manager/tasks/manage_vms.yml

```diff
@@ -173,7 +173,7 @@
     cmd: |-
       test -d /home/zuul && exit 0;
       set -xe -o pipefail;
-      echo "{{ vm_ip.nic.host }}" | sudo tee /etc/hostname;
+      echo "{{ vm_ip.nic.host }}.{{ inventory_hostname }}" | sudo tee /etc/hostname;
```

Comment:
this is needed for sushy, since it will use the """fqdn""" - we need to ensure the host knows its own name - especially controller-0.